lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -397,6 +397,8 @@ class MailboxDashBoardController extends ReloadableController {
       _handleRestoreDeletedMessageFailed();
     } else if (failure is GetRestoredDeletedMessageFailure) {
       _handleRestoreDeletedMessageFailed();
+    } else if (failure is GetAllIdentitiesFailure) {
+      _removeComposerCacheOnWeb();
```

Comment:
IMO, we don't should remove composer cache when can not get identity.

Reply:
Removed